src/openapi_server/test/conftest.py

```diff
@@ -40,5 +40,17 @@ def _(seq, threshold, *args, **kwargs):
     return _
 
 
+@fixture
+def variant_search(make_request):
```

Comment:
I have not seen the tests for variant search. Have you forgotten to commit the test file?